bundles/org.openhab.binding.ojelectronics/src/main/java/org/openhab/binding/ojelectronics/internal/ThermostatHandler.java

```diff
@@ -146,59 +168,65 @@ public void handleThermostatRefresh(Thermostat thermostat) {
         return currentThermostat;
     }
 
-    private void updateManualSetpoint(Thermostat thermostat) {
+    private ThermostatModel getCurrentThermostat() {
+        return Objects.requireNonNull(currentThermostat);
```

Comment:
Per annotation currentThermostat is not null. How could it be here?

Reply:
This is attotated with nullable because it is only set when the RefreshService delivers the first data. However: I changed the behavior. Now the thing goes online when first data arrives.